flang/lib/Evaluate/common.cpp

```diff
@@ -11,8 +11,11 @@
 
 using namespace Fortran::parser::literals;
 
+
```

Comment:
why are these new blank lines necessary?

Reply:
Removed whitespace change. Thank you.